lib/core.js

```diff
@@ -19,18 +19,19 @@ import {remarkMdx} from './remark-mdx.js'
  * @typedef {import("./recma-jsx-rewrite").RecmaJsxRewriteOptions} RecmaJsxRewriteOptions
  *
  * @typedef BaseProcessorOptions
- * @property {boolean} [jsx]
- * @property {PluggableList} [recmaPlugins]
- * @property {PluggableList} [remarkPlugins]
- * @property {PluggableList} [rehypePlugins]
- * @property {boolean} [_contain]
+ * @property {boolean} [jsx=false] Whether to keep JSX
+ * @property {PluggableList} [recmaPlugins] List of recma (esast, JavaScript) plugins
```

Comment:
```suggestion
 * @property {PluggableList} [recmaPlugins] List of recma (estree, JavaScript) plugins
```
Is `esast` the same as `estree`? If so, it may be good to use `estree` as folks are likely more familiar with the term.

Reply:
See my review, which I posted roughly when you reviewed ;)